apps/src/templates/sectionAssessments/MultipleChoiceAnswerCell.jsx

```diff
@@ -24,10 +24,26 @@ class MultipleChoiceAnswerCell extends Component {
   static propTypes = {
     percentValue: PropTypes.number.isRequired,
     isCorrectAnswer: PropTypes.bool,
+    textAnswered: PropTypes.string,
   };
 
   render() {
-    const {percentValue, isCorrectAnswer} = this.props;
+    const {percentValue, isCorrectAnswer, textAnswered} = this.props;
+    if (textAnswered) {
+      return (
+        <div>
+          <div>
+            {textAnswered}
+          </div>
+          <div style={styles.icon}>
+          {isCorrectAnswer &&
+            <FontAwesome icon="check-circle" style={styles.icon}/>
```

Comment:
I think you already have plan for this, but you can restructure your conditionals to avoid the duplication on lines 40 and 59.